Reconstruction/RecTracker/options/TrickTrackReco.py

```diff
@@ -0,0 +1,191 @@
+import os
+import argparse
+from GaudiKernel import SystemOfUnits as units
+from GaudiKernel import PhysicalConstants as constants
+from Gaudi.Configuration import *
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--inputfile', type=str, default='muons_for_seeding.root', help='specify an input file')
+parser.add_argument('--pileupfiles',nargs="+", type=str, default='', help='specify an input file')
+parser.add_argument('--outputfile', type=str, default='muons_for_seeding_tracks.root', help='specify an output file')
+parser.add_argument('--nevents', type=int, default=10, help='specify number of events to process')
+parser.add_argument('--npileup', type=int, default=0, help='specify number of events to process')
+parser.add_argument('--noSignal', action="store_true",  help='use old collection names')
+parser.add_argument('--cleanHits', action="store_true",  help='use old collection names')
+parser.add_argument('--legacyCollectionNames', action="store_true",  help='use old collection names')
```

Comment:
maybe you put legacyCollectionNames and overlayCollectionNames in a mutually exclusive group?

Reply:
Removed those arguments alltogether. Was there only because I had some old samples  I was doing checks on, and it saved time. However, we should maybe take a look at the branch names we are using in the examples and set some conventions - I believe in the FCC Sim Jobs this is done better (`TrackerPositionedHits` vs. `positionedHits`)